pandas/tests/io/formats/test_to_html.py

```diff
@@ -235,6 +235,27 @@ def test_to_html_truncate(datapath):
     assert result == expected
 
 
+@pytest.mark.parametrize(
+    "df,msg,size",
```

Comment:
is parameterisation of `df` and `msg` necessary?

Reply:
is was thinking, more like..

```python
@pytest.mark.parametrize("size", [1, 5])
def test_html_invalid_formatters_arg_raises(size):
    # gh-28469
    df = DataFrame(columns=["a", "b", "c"])
    msg = "Formatters length({}) should match DataFrame number of columns(3)"
    with pytest.raises(ValueError, match=re.escape(msg.format(size))):
        df.to_html(formatters=["{}".format] * size)
```

Reply:
Done!